main.cpp

```diff
@@ -302,6 +256,20 @@ You will define these 5 other 'property' UDTs in Part 1d.
     When you choose your 5 smaller parts, remember that each of 
     these 5 Sub Objects will need to be defined with 5 primitive 
     properties and 3 actions EACH.
+    
+    myUDT: Synthesizer
+
+    A Synthesizer is made up of the following UDT
+        Oscillator
+        LFO
+        Envelope
+        Filter    
+        Keybed
+
+    A Synthesizer can do three things
+        make sound
+        sequence patterns
+        save a patch
```

Comment:
Delete this.  I'm not even sure why you wrote this here, as the instructions did not specify to.
- [ ] revise